feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -36,12 +36,29 @@ import {
   tap,
 } from 'rxjs/operators';
 
+export interface AddedToCartDialogComponentData {
+  productCode: string;
+  quantity: number;
+  /**
+   * Number of cart entries before addToCart was triggered.
+   * @deprecated since 2211.24. Enable feature toggle 'adddedToCartDialogDrivenBySuccessEvent'
+   * and use attribute addedEntryWasMerged instead.
+   */
+  numberOfEntriesBeforeAdd?: number;
+  pickupStoreName?: string;
+  /**
+   * Was the product added to the cart merged into an existing cart entry (with increased quantity),
+   * or did the system create a new cart entry?
```

Comment:
nitpick: The question mark in the end of the sentence is a bit confusing. What about:
```suggestion
   * Tells whether the product added to the cart was merged into an existing cart entry (with increased quantity),
   * or the system created a new cart entry.
```